index.html

```diff
@@ -648,6 +653,28 @@
 &lt;/script&gt;
 </pre>
       </section>
+      <section>
+        <h3>
+          Passing locale information with a message
+        </h3>
+        <pre class="example highlight">
+&lt;!-- controller.html --&gt;
+&lt;script&gt;
+  connection.send("{string: 'Hello, world!', lang: 'en-US'}");
```

Comment:
I would like to suggest including CJK example to more clearly indicate the case the locale could affect font rendering:

```
switch(navigator.language.toLowerCase()) {
  case: 'zh-cn':
    connection.send("{string: '你好，世界!', lang: 'zh-CN'}");
    break;
  case 'ja':
    connection.send("{string: 'こんにちは、世界!', lang: 'ja'}");
    break;
  case 'ko':
    connection.send("{string: '안녕하세요, 세계!', lang: 'ko'}");
    break;
  default:
    connection.send("{string: 'Hello, world!', lang: 'en-US'}");
    break;
}
```
(Note: This example includes all of CJK languages, however, one of them as well as English might be sufficient.)